### PR TITLE
Bug 2090774: Add Readme to plugin folder

### PR DESCRIPTION
**Pull request by aneeshkp**
documentation to explain how ptp plugin works are added to readme under plugin folder

Signed-off-by: Aneesh Puttur <aneeshputtur@gmail.com>

### CODE REVIEW

File: plugins/README.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# Plugins 
+
+In the cloud event proxy framework container, go plugins are utilized to modify the generic event framework by adding domain specific business logics.
+
+Following flags enable plugins.
+1. To load a PTP plugin, specify PTP PLUGIN=true.
```
[the remark this answers is not in the document]
Comment (aneeshkp) [reply plugins/README.md]: Done

File: plugins/README.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  "resource": "/cluster/node/NODE_NAME/sync/ptp-status/ptp-clock-class-change"
+ }
+]
+
+
+```
+2. Listens to linuxptp daemon logs of ptp4l and phc2sys
+3. Generates Events for PTP by parsing ptp4l/phc2sys logs
+4. Add Prometheus metrics for all  PTP events which are accessible within the cloud-event-proxy using 
+5. curl localhost:9091/metrics
```
[the remark this answers is not in the document]
Comment (aneeshkp) [reply plugins/README.md]: Done

**Review by jzding — APPROVED** [review]
/lgtm

**Pull request merged**